app/src/main/java/com/cornellappdev/scoop/ui/screens/HomeScreen.kt

```diff
@@ -51,6 +70,29 @@ fun HomeScreen(
             }
         }
     ) {
+
+        val state = rememberLazyListState()
+
+        LazyColumn(
+            state = state,
+            verticalArrangement = Arrangement.spacedBy(10.dp),
+            modifier = Modifier.padding(16.dp)
+        ) {
+
+            when (rideApiResponse) {
+                is ApiResponse.Error -> items(emptyList<Ride>()) { item ->
+                    RideCard(item)
+                }
+
+                is ApiResponse.Pending -> items(emptyList<Ride>()) { item ->
+                    RideCard(item)
+                }
```

Comment:
Are these two doing anything? Seems like they're just making empty lists and iterating over nothing. If that's the case, you can just turn this when into like an `if` instead and only show the items if it's a success.